openthread/Kconfig

```diff
@@ -100,8 +100,20 @@ config OPENTHREAD_NORDIC_LIBRARY_MTD
 	select OPENTHREAD_SNTP_CLIENT
 	help
 	  Nordic Semiconductor optimized OpenThread features for MTD.
+
+config OPENTHREAD_USER_CUSTOM_LIBRARY
+	bool "User custom configuration"
+	help
+	  No extra features selected.
+
 endchoice
 
+config OPENTHREAD_STRIP_LIBRARIES_BEFORE_INSTALL
```

Comment:
nothing wrong, but you could have chosen to make it possible to strip the libraries independently of install.

Meaning, user only has to decide if stripped version should also be made available, like is done for Zephyr.
And then install will choose to install what is available.
But I leave the decision to you.

Reply:
I agree that it would be the best solution, nevertheless it would require changes in zephyr for what is quite late concerning freeze code before release. I'll stick with current solution.

Reply:
> nevertheless it would require changes in zephyr for what is quite late concerning freeze code before release. 

No, made a proposal here:
https://github.com/lmaciejonczyk/sdk-nrfxlib/pull/2

without stripping:
```
$ cmake -GNinja -DBOARD=nrf52840dk_nrf52840 -DEXTRA_CFLAGS="-Werror" -DCONFIG_OPENTHREAD_BUILD_OUTPUT_STRIPPED=n -Bbuild samples/openthread/cli/
...
$ ninja -Cbuild
...
$ find build . -name 'libopenthread*'
build/modules/openthread/build/src/cli/libopenthread-cli-ftd.a
build/modules/openthread/build/src/core/libopenthread-ftd.a
build/zephyr/subsys/net/lib/openthread/platform/libopenthread_platform.a
```
and with stripping:
```
$ cmake -GNinja -DBOARD=nrf52840dk_nrf52840 -DEXTRA_CFLAGS="-Werror" -Bbuild -DCONFIG_OPENTHREAD_BUILD_OUTPUT_STRIPPED=y samples/openthread/cli/
...
$ ninja -Cbuild
...
$ find build . -name 'libopenthread*'
build/modules/openthread/build/src/cli/libopenthread-cli-mtd.a
build/modules/openthread/build/src/cli/libopenthread-cli-mtd.a.strip
build/modules/openthread/build/src/cli/libopenthread-cli-ftd.a
build/modules/openthread/build/src/cli/libopenthread-cli-ftd.a.strip
...
```

Reply:
Note: I disabled stripping per default in my proposal, as that will significantly reduce build time.

Reply:
Very nice, thank you.